README.md

```
@@ -1,424 +1,109 @@
+| `/whiteboard` | When screen layout, structure, flow, or comparison still feels too ambiguous in text alone — start drawing on the canvas together. |
+| `/whiteboard-coauthoring` | A structured loop for evolving diagrams with the agent: gather context, structure frame by frame, run fresh-viewer tests on what you draw. |
+| `/whiteboard-audit` | Audit existing workspaces — detect orphaned workspaces, tombstone-heavy canvases, and cache/disk mismatches; report cleanup candidates. |
 
-Regression detection after changes follows three layers:
+## Example transcript
 
-```bash
-pnpm test        # unit tests (< 3s, 460+ tests)
-pnpm typecheck   # tsc --noEmit (< 10s)
-pnpm smoke:e2e   # launch MCP stdio subprocess and verify canvas_create -> checkpoint -> viewport/export no_client -> canvas_export_json round trip (< 10s, no API quota)
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)